apps/blog/components/Article/AllArticle/index.tsx

```diff
@@ -125,8 +127,15 @@ const AllArticle = () => {
     }
   }
 
+  const handleScrollToArticleSection = () => {
```

Comment:
shall we wrap this function using `useCallback` and put it as one of the dependencies of the `useEffect` hook above?